cmd/kluctl/commands/cmd_list_images.go

```diff
@@ -14,6 +15,7 @@ type listImagesCmd struct {
 	args.OutputFlags
 	args.RenderOutputDirFlags
 
+	Sort   bool `group:"misc" help:"Sort Output of the image list"`
```

Comment:
I would prefer to just unconditionally sort the output.

Reply:
done in https://github.com/kluctl/kluctl/pull/27/commits/abea27a4892034a8ade1a17a2cf88e55bddce065